.github/pull_request_template.md

```diff
@@ -1,10 +1,42 @@
+## Description
+
+* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
```

Comment:
I feel like instead of `feature`, maybe ask if there is a change to an existing notebook vs. the addition of a new tutorial (and in the latter case if the new tutorial introduces a new feature ->  link to that feature in main `tobac` repo)?

Reply:
Nice idea!